[2023-11-12T08:02:12.186+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v2.create_postgres_table manual__2023-11-12T07:58:06.219731+00:00 [queued]>
[2023-11-12T08:02:12.238+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v2.create_postgres_table manual__2023-11-12T07:58:06.219731+00:00 [queued]>
[2023-11-12T08:02:12.240+0000] {taskinstance.py:1361} INFO - Starting attempt 3 of 6
[2023-11-12T08:02:12.393+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2023-11-12 07:58:06.219731+00:00
[2023-11-12T08:02:12.420+0000] {standard_task_runner.py:57} INFO - Started process 9239 to run task
[2023-11-12T08:02:12.490+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v2', 'create_postgres_table', 'manual__2023-11-12T07:58:06.219731+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/dag-posgres-operator.py', '--cfg-path', '/tmp/tmp04qd0pr2']
[2023-11-12T08:02:12.500+0000] {standard_task_runner.py:85} INFO - Job 83: Subtask create_postgres_table
[2023-11-12T08:02:12.864+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v2.create_postgres_table manual__2023-11-12T07:58:06.219731+00:00 [running]> on host aac4306c1949
[2023-11-12T08:02:13.180+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='nguyenthung' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v2' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2023-11-12T07:58:06.219731+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-12T07:58:06.219731+00:00'
[2023-11-12T08:02:13.180+0000] {sql.py:274} INFO - Executing: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        
[2023-11-12T08:02:13.196+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-11-12T08:02:13.210+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-11-12T08:02:13.240+0000] {sql.py:418} INFO - Running statement: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        , parameters: None
[2023-11-12T08:02:13.279+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v2, task_id=create_postgres_table, execution_date=20231112T075806, start_date=20231112T080212, end_date=20231112T080213
[2023-11-12T08:02:13.343+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-12T08:02:13.394+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
